docs/contributing/release_process.md

```diff
@@ -32,22 +32,30 @@ The 10,000 foot view of the release process is that you prepare a release PR and
 publish a [GitHub Release]. This triggers [release automation](#release-workflows) that
 builds all release artifacts and publishes them to the various platforms we publish to.
 
+We now have a `release.py` script to help with cutting the release PRs.
+
+- `python3 release.py --help` is your friend.
+- `release.py` has only been tested in Python 3.12 (so get with the times :D)
+
 To cut a release:
 
 1. Determine the release's version number
    - **_Black_ follows the [CalVer] versioning standard using the `YY.M.N` format**
      - So unless there already has been a release during this month, `N` should be `0`
    - Example: the first release in January, 2022 → `22.1.0`
+   - `release.py` will calculate this and log to stderr for you copy paste pleasure
 1. File a PR editing `CHANGES.md` and the docs to version the latest changes
+   - Run `python3 release.py [--debug]` to generate most changes
+     - Sub headings in the template, if they have no bullet points need manual removal
+       _PR welcome to improve :D_
+1. If `release.py` fail manually edit; otherwise, yay, skip this step!
    1. Replace the `## Unreleased` header with the version number
-   1. Remove any empty sections for the current release
-   1. (_optional_) Read through and copy-edit the changelog (eg. by moving entries,
+   2. Remove any empty sections for the current release
+   3. (_optional_) Read through and copy-edit the changelog (eg. by moving entries,
```

Comment:
Let's leave the "1." in place, that way the renderer renders the correct numbers and we don't have to worry about keeping them up to date
```suggestion
   1. Remove any empty sections for the current release
   1. (_optional_) Read through and copy-edit the changelog (eg. by moving entries,
```